frontend/src/components/UserDetails.vue

```diff
@@ -116,19 +116,19 @@
 
 <script>
 import UsersRepository from "@/api/UsersRepository";
-import organizations from "@/assets/organizations";
+import app_config from '@/loadconfig';
 import clients from "@/api/ClientsRepository";
 
 export default {
   name: "UserDetails",
   props: ['userId', 'updateOrCreate'],
-  organizations: organizations.map((item) => {
-    item.value = JSON.stringify(item);
-    item.text = `${item.id} - ${item.name}`;
-    return item;
-  }),
   data() {
     return {
+      organizations: app_config.organizations.map((item) => {
+        item.value = JSON.stringify(item);
+        item.text = `${item.id} - ${item.name}`;
+        return item;
+      }),
```

Comment:
If I don't move this to the `data` section of the component, the organizations are empty. I don't really understand it, but it must be something to do with the Vue.js component life-cycle. Note that the `UserSearch` component already had `organizations` in `data`, so this change makes it consistent.